kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml

```diff
@@ -112,4 +117,61 @@
 	
 	<bean id="disbursementVoucherDao" class="edu.arizona.kfs.fp.dataaccess.impl.DisbursementVoucherDaoOjb" parent="platformAwareDao" />
 
+	 <bean id="procurementCardHolderJob" parent="unscheduledJobDescriptor">
```

Comment:
Here to the end of the file was the first merge conflict.  I didn't have to change anything except removing the characters that Git added like <<<<<<< HEAD and =======.